src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+type Props = {
+    journeys: JourneyParams[];
+};
+
+export function getJourneyUrl(journey: JourneyParams) {
+    return `/?from=${journey["fromStationId"]}&to=${journey["toStationId"]}&day=${journey["day"]}&time=${journey["time"]}`;
+}
+
+function getClass(time: JourneyParams) {
+    const times = time["time"];
+    if (times === undefined) {
+        return styles.midday;
+    } else if (times >= 18000 && times <= 39600) {
```

Comment:
Possibly helpful — we have some constants:
```ts
const MINUTE = 60;
const HOUR = 3600;
```
defined in `time.ts` that you can import here — these are helpful for writing out hard-coded times in a readable way. For instance `39600` becomes `11 * HOUR` and it's much clearer at a glance what time this refers to.